plugins/azure/fix_plugin_azure/resource/compute.py

```diff
@@ -3344,6 +3383,11 @@ def pre_process(self, graph_builder: GraphBuilder, source: Json) -> None:
         self.location = graph_builder.location.name if graph_builder.location else ""
 
 
+@define(eq=False, slots=False)
+class AzureVirtualMachineScaleSetInstance(AzureVirtualMachineBase):
```

Comment:
```suggestion
class AzureVirtualMachineScaleSetInstance(AzureVirtualMachineBase):
   # note: instances are collected as part of collecting AzureVirtualMachineScaleSets
```
Consider adding a comment.